jekyll/_cci2/build-config-cookbook.md

```diff
@@ -0,0 +1,334 @@
+
+```yaml
+version: 2.1
+orbs:
+  aws-cli: circleci/aws-cli@0.1.4
+  aws-ecs: circleci/aws-ecs@0.0.3
+jobs:
+  verify-deployment:
+    docker:
+      - image: 'circleci/python:3.7.1'
+    steps:
+      - aws-cli/install
+      - aws-cli/configure:
+          aws-access-key-id: $AWS_ACCESS_KEY_ID
+          aws-region: $AWS_REGION
+      - run:
+          name: Get last task definition
+          command: >
+            TASK_DEFINITION_ARN=$(aws ecs describe-task-definition \
+                --task-definition ${MY_APP_PREFIX}-service \
+                --output text \
+                --query 'taskDefinition.taskDefinitionArn')
+            echo "export TASK_DEFINITION_ARN='${TASK_DEFINITION_ARN}'" >>
+            $BASH_ENV
+      - aws-ecs/verify-revision-is-deployed:
+          family: '${MY_APP_PREFIX}-service'
+          cluster-name: '${MY_APP_PREFIX}-cluster'
+          task-definition-arn: '${TASK_DEFINITION_ARN}'
+workflows:
+  test-workflow:
+    jobs:
+      - verify-deployment
+```
+
+This example illustrates how you can use the orb to install and configure the AWS CLI, retrieve the task definition, and then verify the revision has been deployed.
+
+For more detailed information about the CircleCI Amazon ECS/ECR orb, refer to the [CircleCI Orb Registry](https://circleci.com/orbs/registry/orb/circleci/aws-ecs).
+
+## Deploying Software Changes to Google Kubernetes Engine (GKE)
+
+The Google Kubernetes Engine (GKE) enables you to automate CI/CD strategies to quickly and easily deploy code and application updates to your customers without requiring significant time to deliver these updates. Using the GKE, CircleCI has leveraged this technology, along with development of a GKE-specific CircleCI orb, to enable you to interact with GKE within a specific job. Before working with GKE, you may wish to read Google's technical documentation, which can be found on the [GKE](https://cloud.google.com/kubernetes-engine/docs/) documentation page.
+
+### Prerequisites
+
+#### Setting Environment Variables
+The following environment variables need to be set in CircleCI either directly or via a context:
+
+- `GCLOUD_SERVICE_KEY` (required)
+- `GOOGLE_PROJECT_ID`
+- `GOOGLE_COMPUTE_ZONE`
+
+If you need more information on how to set these environment variables, refer to the [Using Environment Variables](https://circleci.com/docs/2.0/env-vars/) page in the CircleCI documentation.
+
+#### Configuring Your Environment for the CircleCI Platform and Orbs
+
+1) Use CircleCI version 2.1 at the top of your `.circleci/config.yml` file.
+
+`version: 2.1`
+
+If you do not already have pipelines enabled, you'll need to go to **Project Settings -> Advanced Settings** and enable pipelines.
+
+2) Add the orbs stanza below your version, invoking the following orbs:
+
+```orbs:
+    gcloud: circleci/gcp-cli@1.0.6
+    gcr: circleci/gcp-gcr@0.0.2
+    k8s: circleci/kubernetes@0.1.0
+```
```

Comment:
This is the requirements for the orb itself to work. Someone using the orb doesn't need all of these. Just `circleci/gcp-gke`.

Reply:
fixed